app-audiodemo/synth/libsynth.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stddef.h>
+
+#include "mach_defines.h"
+#include "mach_interrupt.h"
+#include "cache.h"
+
+/* Register definitions */
+#define SYNTH_VOICE_CTRL_ENABLE		(1 << 0)
+#define SYNTH_VOICE_CTRL_PHASE_ZERO	(1 << 1)
+#define SYNTH_VOICE_CTRL_SAWTOOTH	(0 << 2)
+#define SYNTH_VOICE_CTRL_TRIANGLE	(1 << 2)
+#define SYNTH_VOICE_CTRL_PULSE		(2 << 2)
+#define SYNTH_VOICE_CTRL_SUBOCTAVE	(3 << 2)
+#define SYNTH_VOICE_CTRL_WT4K_FWD	(4 << 2)
+#define SYNTH_VOICE_CTRL_WT4K_FWD_REV	(5 << 2)
+#define SYNTH_VOICE_CTRL_WT2K_FWD	(6 << 2)
+#define SYNTH_VOICE_CTRL_WT2K_FWD_REV	(7 << 2)
```

Comment:
Those are outdated, use the new #define from my synthtest app

Reply:
Got em.  Can you check, though?